modules/generate-network-policies-ng20.adoc

```diff
@@ -8,14 +8,23 @@
 [role="_abstract"]
 {product-title-short} lets you automatically generate network policies based on the actual observed network communication flows in your environment.
 
-You can generate network policies from the network graph.
-
-The generated policies apply to all of the deployments that exist in the currently-selected cluster and they allow all network traffic observed during the baseline discovery period.
+You can generate policies based on the cluster, namespaces, and deployments that you have selected in the network graph. Policies are generated for any deployments that are included in the current Network Graph scope. For example, the current scope could include the entire cluster, a cluster and namespaces, or individually selected deployments in the selected namespaces. You can also further reduce the scope by applying one of the filters from the *Filter deployments* field with any combination of the cluster, namespace, and deployment selections. For example, you could narrow the scope to deployments in a specific cluster and namespace that are affected by a specific CVE. Policies are generated from the traffic observed during the baseline discovery period.
 
 .Procedure
 . In the {product-title-short} portal, navigate to *Network Graph*.
 . Select a cluster, and then select one or more namespaces.
-. In the network graph header, select *Simulate network policy*. {product-title-short} generates policies for all of the deployments that exist in the selected cluster.
-. Optional: In the information panel that opens, select *Exclude ports & protocols* if you do not want ports and protocols to be scoped in {product-title-short} generated policies.
-. Select *Generate and simulate network policies*.
-The generated network policy configuration YAML file opens in the same panel, and the network graph shows the effects of the policies.
+. Optional: Select individual deployments to restrict the policy generated to just those deployments. You can also use the *Filter deployments* feature to further narrow the scope.
+. In the network graph header, select *Network policy generator*.
+. Optional: In the information panel that opens, select *Exclude ports & protocols* to remove the port/protocol restrictions when generating network policies from a baseline. For example, the `nginx3` deployment makes a port 80 connection to `nginx4`, and this is included as part of the baseline for nginx4. If policies are generated and this checkbox is not selected (the default behavior), the generated policy will restrict the allowed connections from `nginx3` to `nginx4` to only port 80. If policies are generated with this option selected, the generated policy will allow any port in the connection from `nginx3` to `nginx4`.
```

Comment:
Also, this feels like a lot of additional context to add after an instruction. Is it worth taking everything starting from "For example.." and making it a separate paragraph that's still a part of this step? I feel like it would make the instruction more scannable and allow users to skip the example if they don't need one.